Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 13
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2444
"GET /about HTTP/1.1" 200 8
"GET / HTTP/1.1" 200 13
"GET / HTTP/1.1" 200 13
"GET / HTTP/1.1" 200 13
"GET /about HTTP/1.1" 200 8
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /
"GET / HTTP/1.1" 404 2396
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 37
"GET /about HTTP/1.1" 200 55
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 37
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2443
"GET /about HTTP/1.1" 200 80
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py", line 5, in main
    return render(request, "index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
"GET / HTTP/1.1" 500 79346
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2443
"GET /about HTTP/1.1" 200 55
Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py", line 5, in main
    return render(request, "index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: index.html
"GET / HTTP/1.1" 500 79471
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homeworks\settings.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py", line 5, in main
    return render(request, "index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loaders\base.py", line 23, in get_template
    contents = self.get_contents(origin)
               ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loaders\cached.py", line 26, in get_contents
    return origin.loader.get_contents(origin)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loaders\filesystem.py", line 22, in get_contents
    with open(origin.name, encoding=self.engine.file_charset) as fp:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: [Errno 22] Invalid argument: 'C:\\Users\\zeidm\\My_projects\\django_homeworks\\homeworks\\homework_1\templates\\homework_1\\index.html'
"GET / HTTP/1.1" 500 110696
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homeworks\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homeworks\settings.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py", line 5, in main
    return render(request, "index.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loaders\base.py", line 23, in get_template
    contents = self.get_contents(origin)
               ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loaders\cached.py", line 26, in get_contents
    return origin.loader.get_contents(origin)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loaders\filesystem.py", line 22, in get_contents
    with open(origin.name, encoding=self.engine.file_charset) as fp:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: [Errno 22] Invalid argument: 'C:\\Users\\zeidm\\My_projects\\django_homeworks\\homeworks\\homework_1\templates\\homework_1\\index.html'
"GET / HTTP/1.1" 500 110821
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homeworks\settings.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homeworks\settings.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 155
"GET /about HTTP/1.1" 200 55
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 162
"GET / HTTP/1.1" 200 162
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 241
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 295
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2443
"GET / HTTP/1.1" 200 335
"GET /about HTTP/1.1" 200 235
"GET / HTTP/1.1" 200 335
"GET /about HTTP/1.1" 200 235
"GET /about HTTP/1.1" 200 235
"GET /about HTTP/1.1" 200 235
"GET /about HTTP/1.1" 200 235
"GET /about HTTP/1.1" 200 235
Watching for file changes with StatReloader
Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\http\request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Bad Request: /
"GET / HTTP/1.1" 400 71727
Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\http\request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
"GET /favicon.ico HTTP/1.1" 400 71565
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 335
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2443
"GET /about HTTP/1.1" 200 235
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /
"GET / HTTP/1.1" 404 2660
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2711
Not Found: /about
"GET /about HTTP/1.1" 404 2693
Not Found: /client/<int:client_id>/
"GET /client/%3Cint:client_id%3E/ HTTP/1.1" 404 2763
Not Found: /client/<1>/
"GET /client/%3C1%3E/ HTTP/1.1" 404 2727
Not Found: /client/<int:1>/
"GET /client/%3Cint:1%3E/ HTTP/1.1" 404 2739
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\models.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /
"GET / HTTP/1.1" 404 2591
Watching for file changes with StatReloader
Not Found: /
"GET / HTTP/1.1" 404 2591
Not Found: /about
"GET /about HTTP/1.1" 404 2624
"GET /clients HTTP/1.1" 301 0
"GET /clients/ HTTP/1.1" 200 491
"GET /products/ HTTP/1.1" 200 493
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
"GET /clients/ HTTP/1.1" 200 491
"GET /products/ HTTP/1.1" 200 493
"GET /orders/ HTTP/1.1" 200 489
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
"GET /orders/ HTTP/1.1" 200 489
"GET /ordered_products/ HTTP/1.1" 200 886
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
"GET /orders/ HTTP/1.1" 200 687
"GET /ordered_products/ HTTP/1.1" 200 886
Watching for file changes with StatReloader
Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\http\request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Bad Request: /
"GET / HTTP/1.1" 400 71710
Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\http\request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Bad Request: /
"GET / HTTP/1.1" 400 71710
Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\http\request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
"GET /favicon.ico HTTP/1.1" 400 71547
Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\http\request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
"GET /favicon.ico HTTP/1.1" 400 71547
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2993
"GET /orders/ HTTP/1.1" 200 687
"GET /ordered_products/ HTTP/1.1" 200 886
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homeworks\settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
"GET / HTTP/1.1" 200 696
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2988
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2988
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2988
Internal Server Error: /products/
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 329, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: homework_1_product.photo

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py", line 17, in product_list
    return render(request, 'products.html', {'products': products})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\defaulttags.py", line 198, in render
    len_values = len(values)
                 ^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 329, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: homework_1_product.photo
"GET /products/ HTTP/1.1" 500 199357
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
Internal Server Error: /products/
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py", line 17, in product_list
    return render(request, 'products.html', {'products': products})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'product_create' not found. 'product_create' is not a valid view function or pattern name.
"GET /products/ HTTP/1.1" 500 148107
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
"GET /products/ HTTP/1.1" 200 690
Internal Server Error: /create/
Traceback (most recent call last):
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\homeworks\homework_1\views.py", line 48, in product_create
    return render(request, 'product_create.html', {'form': form})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zeidm\My_projects\django_homeworks\venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: product_create.html
"GET /create/ HTTP/1.1" 500 83492
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
"GET /products/ HTTP/1.1" 200 690
"GET /create/ HTTP/1.1" 200 1233
"GET /clients/ HTTP/1.1" 200 491
"GET /products/ HTTP/1.1" 200 690
"GET /create/ HTTP/1.1" 200 1233
"GET /orders/ HTTP/1.1" 200 687
"GET /ordered_products/ HTTP/1.1" 200 886
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
"GET /clients/ HTTP/1.1" 200 491
"GET /products/ HTTP/1.1" 200 690
"GET /orders/ HTTP/1.1" 200 687
"GET /ordered_products/ HTTP/1.1" 200 886
"GET /clients/ HTTP/1.1" 200 491
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 696
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 3165
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homeworks\settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 9699
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 12621
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 13578
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4527
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=SQLPanel HTTP/1.1" 200 115
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=HistoryPanel HTTP/1.1" 200 2298
"GET /static/debug_toolbar/js/history.js HTTP/1.1" 200 3515
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=VersionsPanel HTTP/1.1" 200 621
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=SettingsPanel HTTP/1.1" 200 25966
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=CachePanel HTTP/1.1" 200 1413
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=SignalsPanel HTTP/1.1" 200 2107
"GET /__debug__/history_refresh/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&exclude_history=True HTTP/1.1" 200 1588
"GET /__debug__/history_sidebar/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&exclude_history=True HTTP/1.1" 200 7570
"GET /__debug__/history_sidebar/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&exclude_history=True HTTP/1.1" 200 7570
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=SettingsPanel HTTP/1.1" 200 25966
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=TemplatesPanel HTTP/1.1" 200 6130
"GET /__debug__/template_source/?template=base.html&template_origin=IkM6XFxVc2Vyc1xcemVpZG1cXE15X3Byb2plY3RzXFxkamFuZ29faG9tZXdvcmtzXFxob21ld29ya3NcXGhvbWV3b3JrXzFcXHRlbXBsYXRlc1xcaG9tZXdvcmtfMVxcYmFzZS5odG1sIg:1rwF8Y:9g-Ma30rqc-D_WXDtaQOIpL6gruu5eNTqaSIL3sfB2s HTTP/1.1" 200 1152
"GET /__debug__/template_source/?template=index.html&template_origin=IkM6XFxVc2Vyc1xcemVpZG1cXE15X3Byb2plY3RzXFxkamFuZ29faG9tZXdvcmtzXFxob21ld29ya3NcXGhvbWV3b3JrXzFcXHRlbXBsYXRlc1xcaG9tZXdvcmtfMVxcaW5kZXguaHRtbCI:1rwF8Y:hb78f1VEwhdyZcAcxb3qEoHzTIR9VUVlSJN87KHOgGM HTTP/1.1" 200 901
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=RequestPanel HTTP/1.1" 200 1560
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=SQLPanel HTTP/1.1" 200 115
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=StaticFilesPanel HTTP/1.1" 200 34081
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=CachePanel HTTP/1.1" 200 1413
"GET /__debug__/render_panel/?store_id=28c2c9dbe4bd4c658f07e885dede18f3&panel_id=SignalsPanel HTTP/1.1" 200 2107
"GET /clients/ HTTP/1.1" 200 9895
"GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 304 0
"GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
"GET /static/debug_toolbar/css/print.css HTTP/1.1" 304 0
"GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
"GET /orders/ HTTP/1.1" 200 10087
"GET /orders/ HTTP/1.1" 200 10087
"GET /__debug__/render_panel/?store_id=8318016edb0c40b5887d580ec333c390&panel_id=ProfilingPanel HTTP/1.1" 200 35538
C:\Users\zeidm\My_projects\django_homeworks\homeworks\homeworks\settings.py changed, reloading.
